xml/System.Web.Services.Description/SoapProtocolImporter.xml

```diff
@@ -293,7 +293,7 @@
         <param name="uriList">A space-delimited list of URIs.</param>
         <summary>Checks for the presence of "http://schemas.xmlsoap.org/soap/encoding/" in a string that represents a list of Uniform Resource Indicators (URIs).</summary>
         <returns>
-          <see langword="true" /> it the string contains [http://schemas.xmlsoap.org/soap/encoding/](http://schemas.xmlsoap.org/soap/encoding/); otherwise <see langword="false" />.</returns>
+          <see langword="true" /> it the string contains "http://schemas.xmlsoap.org/soap/encoding/"; otherwise <see langword="false" />.</returns>
```

Comment:
This isn't 'semantically' a link, so I removed it to be consistent with the summary section above. Let me know if I should put it back! 🙂

Reply:
This still renders as a link. I think the best choice here is \<see langword="http://schemas.xmlsoap.org/soap/encoding/" />.